client/components/jetpack/daily-backup-status/use-get-display-date.js

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
I created a custom hook to give easy access to date/time formatting. Turns out it requires quite a few dependencies, so encapsulating it all like this minimizes extra noise in the rest of the DailyBackupStatus components.